lib/rover/data_frame.rb

```diff
@@ -532,7 +540,7 @@ def join(other, how:, on: nil)
         else
           matches.each do |r2|
             keys.each do |k|
-              vectors[k] << (r2[k] || r[k])
+              vectors[k] << (r2[k].nil? ? r[k] : r2[k])
```

Comment:
That's almost exactly the same thing isn't it? Except that if the value is `false` it will take the `r2[k]` case, which it wouldn't do in the previous version. Was this the intention?

Reply:
Precisely